db/db_compaction_test.cc

```diff
@@ -9434,104 +9434,112 @@ TEST_F(DBCompactionTest, CompactionWithChecksumHandoffManifest2) {
 }
 
 TEST_F(DBCompactionTest, FIFOChangeTemperature) {
-  for (bool write_time_default : {false, true}) {
-    SCOPED_TRACE("write time default? " + std::to_string(write_time_default));
+  for (bool should_allow_trivial_copy : {false, true}) {
```

Comment:
We should check if a compaction is actually ran. Can do so by checking stats like `COMPACTION_CPU_TOTAL_TIME`